### PR TITLE
Create contribute_ways.json

### LINKED ISSUE #60: Content - Create content for Ways to Contribute section
## Description
As a user,
I need a brief description for each heading of the following marked section
so that as a newcomer I understand the role of each team.

## Mocks
(develop content for section in red box only)
ways to contribute

[design link](https://xd.adobe.com/view/0bfde3f8-de19-4c3d-61e9-0d11752d875d-7dcc/)

### Update [Required]
Add descriptions for the following
- [ ] ways to contribute (one-liner welcoming text, asking them to check out our teams and other headings below it for more details).
- [ ] code
- [ ] documentation/training
- [ ] design
- [ ] testing
- [ ] outreach/research
- [ ] project management

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3-5 days

### PR DESCRIPTION
**Pull request by nandini45**
### Description
content for contribution page

Fixes #60 

### Type of Change:
**Delete irrelevant options.**

- Documentation

**Code/Quality Assurance Only**
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/content/contribute_ways.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    "sections": [
+      {
+        "title": "Ways to Contribute",
+        "content": [
+
+            {
+                "subtitle": "Code",
+                "content":[
+                    {
+                        "par": "Zulip: https://anitab-org.zulipchat.com/#narrow/stream/216321-coding"
```
Comment (annabauza) [inline src/content/contribute_ways.json:11]: please leave the links separately, I don't think we want it as a part of text

**Review by annabauza — COMMENTED** [review]
@nandini45 can you please move this to js file so if we have any images we can load them using require reference?
Thanks.

**Comment by tichnas** [comment]
Hi, I've a PR (#162) for the Contribution Ways interface. It'll be great if the content file follows the structure as I've used (https://github.com/anitab-org/anitab-org.github.io/pull/162/files#diff-86f38b9c0acf9cf117ba7949ed1a23cb0872b40f74ac3ee7855c3d45bb5e7ac8) as the code largely depends on that.

If required, I can also do the changes, ie, merge this content and I'll fetch it and re-format it as required.

Sorry for the inconvenience.
Thanks

**Comment by nandini45** [comment]
@annabauza ok 
@tichnas ah no issue i can restructure it

**Review by annabauza — CHANGES_REQUESTED** [review]
as in comment

File: src/content/contribute_ways.json (hunk 1 above)
Comment (annabauza) [reply src/content/contribute_ways.json:11]: @nandini45 can you please fix it?

**Pull request closed without merging**